src/wombats/scheduler/core.clj

```diff
@@ -44,3 +46,62 @@
 ;; TODO #349
 ;; Schedule active & active intermission games
 ;; in the case of system restart.
+
+(def add-game-request
+  {:arena-id "41193bec-4cf3-4f5d-8386-aed3ae3e5745"
+   :game
+   {:game/status :pending-open
+    :game/num-rounds 8
+    :game/round-length 120000
```

Comment:
These constants should be in the constants file (so that we can easily change the defaults).